pom.xml

```diff
@@ -40,12 +40,12 @@
             <dependency>
                 <groupId>cz.kb.openbanking.adaa.client</groupId>
                 <artifactId>api</artifactId>
-                <version>1.0.0</version>
+                <version>1.1.0-SNAPSHOT</version>
```

Comment:
Let's make this `1.1.0` version. It's not a `SNAPSHOT` anymore. Is it?

Reply:
We need to add CI/CD tool (Jenkins or Travis CI) to build and release branches. Then after merge to master, we will release version 1.1.0

Reply:
👍